addons/mass_mailing/models/mass_mailing.py

```diff
@@ -620,6 +621,15 @@ def onchange_mass_mailing_campaign_id(self, cr, uid, id, mass_mailing_campaign_i
                    'medium_id': mass_mailing_campaign[0].medium_id.id}
             return {'value': dic}
 
+    def create(self, cr, uid, vals, context=None):
+        if not vals['mass_mailing_campaign_id']:
```

Comment:
For the superuser, it was a mistake. 

For the create method, I think it is necessary, as there are no inheritS on mail.mass_mailing.campaign on a mail.mass_mailing. The goal of the task is to create automatically a mass_mailing_campaign on a mass_mailing.